inc/setup/cachify.hdd.nginx.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/* Quit */
+defined('ABSPATH') OR exit;
+?>
+
+	<table class="form-table">
+		<tr>
+			<th>
+				<?php esc_html_e( 'nginxconf HDD setup', 'cachify' ); ?>
+			</th>
+			<td>
+				<label for="cachify_setup">
+					<?php esc_html_e( 'Please add the following lines to your nginx.conf', 'cachify' ); ?>
+				</label>
+			</td>
+		</tr>
+
+		<tr>
+			<th>
+				<?php esc_html_e( 'Notes', 'cachify' ); ?>
+			</th>
+			<td>
+				<ul style="list-style-type:circle">
+					<li>
+						<?php esc_html_e( 'For domains with FQDN, the variable ${http_host} must be used instead of ${host}.', 'cachify' ); ?>
+					</li>
+				</ul>
+			</td>
+		</tr>
+	</table>
+
+	<div style="background:#fff;border:1px solid #ccc;padding:10px 20px"><pre>
+## GZIP
+gzip_static on;
+
+## CHARSET
+charset utf-8;
+
+## INDEX LOCATION
+location / {
+    if ( $query_string ) {
+        return 405;
+    }
+    if ( $request_method = POST ) {
+        return 405;
+    }
+    if ( $request_uri ~ /wp-admin/ ) {
+        return 405;
+    }
+    if ( $http_cookie ~ (wp-postpass|wordpress_logged_in|comment_author)_ ) {
+        return 405;
+    }
+
+    error_page 405 = @nocache;
+
+    try_files /wp-content/cache/cachify/https-${host}${uri}index.html /wp-content/cache/cachify/${host}${uri}index.html @nocache;
```

Comment:
You mean try files? I didnt test this. I have no nginx server. But this code was posted by swispydy on GitHub in an issue (sry I'm on a mobile device). And the user posted this code on his website so the code should work.
My thought behind all this is we should include standard configs that work for every server with less notes so a standard user is not confused. If someone wants specific code for HTTP or HTTPS only he can look into the wiki. But if we try to include 3 different configs it will confuse all users that are new with such things